tests/backend/test_GameManager.py

```diff
@@ -0,0 +1,34 @@
+from backend.GameManager import *
+import pytest
+
+
+@pytest.fixture
+def game_manager():
+    game_manager = GameManager(7, 9)
+    yield game_manager
+
+class TestGameManager:
+    def test_initial_turn(self, game_manager):
+        # Test if the initial turn is WHITE
+        assert game_manager.get_turn() == 'white'
+
+    def test_all_pieces_placed(self, game_manager):
```

Comment:
This test is only testing the negative path for the `placement_complete` function. We should have the happy path test for if `self.pieces.all_pieces_placed()` is true.